guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -131,33 +131,33 @@ Such servers lie outside of the {Project} infrastructure, however, it is recomme
+By associating Content Views with lifecycle environments, you make content available to hosts in a defined way.
 See xref:figu-Content_Life_Cycle_{context}[] for visualization of the process.
 For a detailed overview of the content management process see {ContentManagementDocURL}Importing_Custom_Content[Importing Custom Content] in _{ContentManagementDocTitle}_.
-The following section provides general scenarios for deploying content views as well as life cycle environments.
+The following section provides general scenarios for deploying content views as well as lifecycle environments.
 
-The default life cycle environment called *Library* gathers content from all connected sources.
+The default lifecycle environment called *Library* gathers content from all connected sources.
 It is not recommended to associate hosts directly with the Library as it prevents any testing of content before making it available to hosts.
-Instead, create a life cycle environment path that suits your content workflow.
+Instead, create a lifecycle environment path that suits your content workflow.
 The following scenarios are common:
 
-* *A single life cycle environment* – content from Library is promoted directly to the production stage.
+* *A single lifecycle environment* – content from Library is promoted directly to the production stage.
 This approach limits the complexity but still allows for testing the content within the Library before making it available to hosts.
 +
-image::life-cycle-path-basic-satellite.png[A single life cycle environment]
+image::life-cycle-path-basic-satellite.png[A single lifecycle environment]
 
-* *A single life cycle environment path* – both operating system and applications content is promoted through the same path.
+* *A single lifecycle environment path* – both operating system and applications content is promoted through the same path.
 The path can consist of several stages (for example *Development*, *QA*, *Production*), which enables thorough testing but requires additional effort.
 +
-image::life-cycle-path-simple-satellite.png[A single life cycle environment path]
+image::life-cycle-path-simple-satellite.png[A single lifecycle environment path]
 
-* *Application specific life cycle environment paths* – each application has a separate path, which allows for individual application release cycles.
-You can associate specific compute resources with application life cycle stages to facilitate testing.
+* *Application specific lifecycle environment paths* – each application has a separate path, which allows for individual application release cycles.
+You can associate specific compute resources with application lifecycle stages to facilitate testing.
 On the other hand, this scenario increases the maintenance complexity.
 +
-image::life-cycle-path-diverged-satellite.png[Application specific life cycle environment paths]
+image::life-cycle-path-diverged-satellite.png[Application specific lifecycle environment paths]
```

Comment:
```suggestion
image::lifecycle-path-diverged-satellite.png[Application specific lifecycle environment paths]
```

I find these images:

````
./guides/doc-Planning_for_Project/images/life-cycle-path-diverged-satellite.png
./guides/doc-Planning_for_Project/images/life-cycle-path-basic-satellite.png
./guides/doc-Planning_for_Project/images/life-cycle-path-simple-satellite.png
./guides/doc-Planning_for_Project/images/life-cycle-satellite.png
./guides/doc-Managing_Content/images/application-life-cycle-satellite.png
./guides/doc-Managing_Content/images/application-life-cycle.png
````

Could you please rename them too?

Reply:
I was able to rename the images in `doc-Planning_for_Project` but I couldn't find the images for `doc-Managing_Content`.

Reply:
Maybe I was duped by symlinks???

Reply:
I found them!